recipes-devtools/python/python3-pip_%.bbappend

```diff
@@ -0,0 +1,3 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+
+SRC_URI += "file://0001-factory.py-disable-unsupported-wheel-checks.patch"
```

Comment:
If there is a bug here we need to fix on original pip recipe and not adding a bbappend file.

Reply:
This is an issue we see only with NXP wheels and upstream doesn't seem to be the issue, so hence putting a bbappend only here in this layer.

Reply:
We need to find another solution as this will change the behavior for everyone and it is not acceptable. Do you have a way for me to reproduce the error?

Reply:
You can simply build the BSP with these recipes included and you will see the errors. I see it on honister, should I do the PR in that branch only instead?

Reply:
We need to fix it without this hack. We need to get to the root cause of the error so we avoid it to happen. 

I'll try to reproduce it here once I have a time window ...